pkg/gen/filters/filterqt/extern.go

```diff
@@ -37,3 +38,15 @@ func qtExtern(xe *model.Extern) QtExtern {
 	  Component: component,
 	}
 }
+
+func qtExterns(ex_list []*model.Extern) []QtExtern {
+	var qtExternsList = []QtExtern {}
+	for _, element := range ex_list { 
+		qtExternsList = append(qtExternsList, qtExtern(element))
+	}
+	return qtExternsList
+}
+
+func qtMakeListOfFields_extern(inputList []QtExtern, fieldName string) ([]string, error){
```

Comment:
I think input should be enough as name. The func name seems to be oddly long. Why make another filter for the list of fields filter?